new-pipeline/src/dist.js

```diff
@@ -1870,3 +1884,24 @@ function saveStateToUrlAndCookie() {
 function isHistogramsListKeyed(histogramsList) {
   return histogramsList.some(entry => entry.title !== "");
 }
+
+function makenewHistogramListCopy(histogramlist) {
+ return histogramlist.map((item) => {
+    return {
+      title: item.title,
+      histograms: item.histograms.map((histogram) => {
```

Comment:
I think we can use some ECMA6 notation here to save us some trouble:

```js
return histogramList.map(item => {
  return {
    ...item,
    histograms: item.histograms.map(histogram => {
      return {
        ...histogram,
        buckets: histogram.buckets.slice(0),
        values: histogram.values.slice(0),
      };
    }),
  };
});
```

This is javascript's [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), and it comes in handy all over the place.